src/http/server.rs

```diff
@@ -34,7 +34,7 @@ async fn graphql_request(req: Request<Body>, server_ctx: &ServerContext) -> Resu
     response = response.set_cache_control(ttl);
   }
 
-  response.to_response()
+  response.to_response(server_ctx.server.set_headers.clone())
```

Comment:
can set headers over here instead of inside of `to_response`

Reply:
Done.